vmm/src/acpi.rs

```diff
@@ -402,6 +402,68 @@ fn create_spcr_table(base_address: u64, gsi: u32) -> Sdt {
     spcr
 }
 
+#[cfg(target_arch = "aarch64")]
+fn create_dbg2_table(base_address: u64) -> Sdt {
+    let ns = b"_SB_.COM1";
+    let ddi_off = 44usize;
```

Comment:
Do you mind expending the variable name a little bit like `debug_device_info_offset` or `dbg_dev_info_off`? That would be easier to review and maintain.

Same comments to `ns`, `ns_off`, `ba_off` and `az_off`.